src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -66,7 +66,12 @@ public CaseSearchResult searchCases(
 
         rejectBlackListedQuery(jsonSearchRequest);
 
-        CrossCaseTypeSearchRequest request = new CrossCaseTypeSearchRequest(caseTypeIds, convertJsonStringToJsonNode(jsonSearchRequest));
+        CrossCaseTypeSearchRequest request = new CrossCaseTypeSearchRequest.Builder()
+            .withCaseTypes(caseTypeIds)
+            .withSearchRequest(convertJsonStringToJsonNode(jsonSearchRequest))
```

Comment:
can we rename convertJsonStringToJsonNode to `stringToJsonNode`?

Reply:
done